packages/amplify-cli/src/init-steps/s9-onSuccess.ts

```diff
@@ -177,6 +179,14 @@ function generateReadMeFile(context: $TSContext) {
   writeReadMeFile(readMeFilePath);
 }
 
+function generateHooksSampleFiles(context: $TSContext) {
+  // TODO: delete amplify/hooks if exist
```

Comment:
I'm not sure we want to do that. What if a customer pulls a sample project from github and then runs `amplify init` on it?